### ISSUE TITLE
meta batch + video combine fails

### ISSUE THREAD

**Issue by dskill** [Kosinkadink/ComfyUI-VideoHelperSuite#276]
I figured out how to use meta batch to load larger videos and process them in sections, which is great. However, when I use meta batch, the video combine node always generates videos of 0 bytes, and I get an error at the very end of my job (after all the meta batches have processed). 

Right now I've added an additional `Save Image` node so that I can just manually stitch the final frames together as a workaround.

**Comment by rocky533** [comment]
Meta batch will generate nothing in video combine until the end of a run of all batches. 

2nd. ummm gifs... i dont know the error TBH.   But save a MP4.  Not gif for batches it does not work with them. I dont think it works for any animated image format.  Use MP4

**Comment by dskill** [comment]
Yeah i was actually using MP4 (h264) so i'm not sure why this message is
complaining about gifs.  Maybe if you use meta batch it ignores the
settings you have in the video combine node or something like that?

**Comment by monster-echo** [comment]
I have the same problem

The question is we combine an image list with the combined video node.  So add an image list to batch node before it . It worked.

**Comment by olir** [comment]
Yes, this works (at least length is correct), but should not connect meta_batch, temp output is generated for each batch and audio copy does not work.
